src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/instructions/instructions.rs

```diff
@@ -1,36 +1,79 @@
 use crate::prelude::*;
 
+use radix_engine::types::MANIFEST_SBOR_V1_MAX_DEPTH;
+use radix_engine_toolkit::functions::address::decode as RET_decode_address;
+
 #[derive(Clone, Debug, PartialEq, Eq, derive_more::Display, uniffi::Record)]
 #[display("{}", self.instructions_string())]
 pub struct Instructions {
-    pub(crate) secret_magic: InstructionsSecretMagic, // MUST be first prop, else you break build.
+    secret_magic: InstructionsSecretMagic, // MUST be first prop, else you break build.
```

Comment:
Hide constructor so that we ensure we always go via `TryFrom<(&[ScryptoInstruction], NetworkID)> ` which calls `scrypto_decompile` this checking SBOR depth